deployments/ansible/roles/collector/tasks/win_install.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+# Install splunk-otel-collector on Windows
+- name: Install Splunk OpenTelemetry Collector with msi package manager for Windows
+  ansible.builtin.import_tasks: otel_win_install.yml
+  when: ansible_os_family == "Windows"
```

Comment:
`when: ansible_os_family == "Windows"` conditions are not needed in this file since it's imported with the same condition